qualtran/bloqs/mod_arithmetic/mod_multiplication_test.py

```diff
@@ -183,11 +183,11 @@ def test_dirtyoutofplacemontgomerymodmul_symbolic_cost(uncompute):
 
     # Litinski 2023 https://arxiv.org/abs/2306.08585
     # Figure/Table 8. Lists modular multiplication as 2.25n^2+9n toffoli.
-    # The following formula is 2.25n^2+8.25n-1 written with rationals because sympy comparison fails with floats.
+    # The following formula is 2.25n^2+7.25n-1 written with rationals because sympy comparison fails with floats.
```

Comment:
@NoureldinYosri Can you please verify that this change is expected? I've simply updated the test assuming the constant factor reduction in qroam uncomputation is translating into reduced constant factors here.

Reply:
yes, the constant factor from qroam gets multiplied by $n/4$ here so it reduces the coefficient of $n$